src/applications/ivc-champva/10-10D/config/submitTransformer.js

```diff
@@ -107,18 +113,18 @@ export default function transformForSubmit(formConfig, form) {
     // If certifier is also applicant, we don't need to fill out the
     // bottom "Certification" section of the PDF:
     certification:
-      transformedData?.certifierRelationship?.relationshipToVeteran
-        ?.applicant === true
+      transformedData?.certifierRole === 'applicant'
         ? { date: currentDate }
         : {
             date: currentDate,
             lastName: transformedData?.certifierName?.last || '',
             middleInitial: transformedData?.certifierName?.middle || '',
             firstName: transformedData?.certifierName?.first || '',
             phoneNumber: transformedData?.certifierPhone || '',
-            relationship: transformRelationship(
-              transformedData?.certifierRelationship,
-            ),
+            // will produce string list of checkboxgroup keys (e.g., "spouse; child")
+            relationship: trueKeysOnly(
+              transformedData?.certifierRelationship?.relationshipToVeteran,
+            ).join('; '),
```

Comment:
Previously, a certifier could have multiple relationships (e.g., spouse AND parent if there were two applicants listed). However, when we filled the PDF we were only showing the first of those values.

Now, we're producing a list like this:
```javascript
relationship: trueKeysOnly(
  // { spouse: true, parent: true, child: false }
).join('; ')
```
Produces:
```javascript
"spouse; parent"
```
which is then stamped in the PDF.